codespell_lib/_codespell.py

```diff
@@ -272,6 +275,26 @@ def open_with_internal(self, filename: str) -> Tuple[List[str], str]:
 
         return lines, encoding
 
+class TomlParser:
```

Comment:
The above example does not handle the case where neither `tomli` nor `tomllib` are available. This could be achieved by nesting another except, but personally I prefer abstracting this logic away from the main code.

Reply:
Well, the existing code does handle that case, doesn't it?

Reply:
Except for `tomlib`-> `tomllib` of course. Just fix that please.